WinUIGallery/WinUIGallery.csproj

```diff
@@ -84,8 +83,6 @@
                       Version="$(CommunityToolkitWinUIVersion)" />                                                
     <PackageReference Update="Microsoft.Graphics.Win2D"
                       Version="$(GraphicsWin2DVersion)" />
-    <PackageReference Update="Microsoft.VCRTForwarders.140"
-                      Version="1.0.6" />
     <!-- Get latest WinRT.Runtime.dll from C#/WinRT -->
     <PackageReference Update="Microsoft.Windows.CsWinRT" Version="2.0.3" />
```

Comment:
should be $(MicrosoftCsWinRTPackageVersion)

Reply:
oh, somehow that change was lost. thanks